packaging/windows/buildpackage.sh

```diff
@@ -47,9 +47,20 @@ function makelauncher()
 	MOD_ID="${3}"
 	PLATFORM="${4}"
 
-	convert "${ARTWORK_DIR}/${MOD_ID}_16x16.png" "${ARTWORK_DIR}/${MOD_ID}_24x24.png" "${ARTWORK_DIR}/${MOD_ID}_32x32.png" "${ARTWORK_DIR}/${MOD_ID}_48x48.png" "${ARTWORK_DIR}/${MOD_ID}_256x256.png" "${BUILTDIR}/${MOD_ID}.ico"
-	install_windows_launcher "${SRCDIR}" "${BUILTDIR}" "win-${PLATFORM}" "${MOD_ID}" "${LAUNCHER_NAME}" "${DISPLAY_NAME}" "${FAQ_URL}"
+	TAG_TYPE="${TAG%%-*}"
+	TAG_VERSION="${TAG#*-}"
+	BACKWARDS_TAG="${TAG_VERSION}-${TAG_TYPE}"
```

Comment:
```suggestion
	TAG_VERSION="${TAG#*-}"
	VERSION="${TAG_VERSION:0:4}.${TAG_VERSION:4:2}.${TAG_VERSION:6:2
```

Reply:
can be done in a followup incase its wanted